### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by loy0516** [Babyhamsta/Aimmy#1501]
Hello, please update the model training tutorial with a more complete version. The YOLOv5 model I trained following the current tutorial cannot work in Aimmy after being converted to the ONNX format.

**Comment by Babyhamsta** [comment]
Please use Yoable our new tool if you are going to label, it's a lot easier to use than makesense.

It can support most v8/v5 onnx models: https://github.com/Babyhamsta/Yoable
It's also being updated and worked on which makesense is not.

**Comment by loy0516** [comment]
from ultralytics import YOLO


def train_model():
    model = YOLO(r'yolov8s.pt')  # 选择适合的预训练模型
    model.train(data=r'data.yaml', epochs=10, save=True)
    model.export(format="onnx")


if __name__ == '__main__':
    train_model()

WHY Not working

![Image](https://github.com/user-attachments/assets/14021481-448e-4683-8cdf-4b1a363920a2)

**Comment by loy0516** [comment]
@Babyhamsta It's not the fault of the annotation tool, but rather the model not meeting the requirements. How can I train a model that meets Aimmy's requirements?
